conditions/operator_conditions.go

```diff
@@ -32,9 +32,15 @@ var (
 	ErrNoOperatorCondition = fmt.Errorf("operator Condition CRD is nil")
 )
 
-// TODO: verify from OLM if this will be the name of the environment variable
-// which is set for the Condition resource owned by the operator.
-const operatorCondEnvVar = "OPERATOR_CONDITION_NAME"
+const (
+	// operatorCondEnvVar is the env variable which
+	// contains the name of the Condition CR associated to the operator,
+	// set by OLM.
+	operatorCondEnvVar = "OPERATOR_CONDITION_NAME"
+	// OperatorNsEnvVar is the env variable which can be used to set the namespace
+	// of the operator if running locally.
+	OperatorNsEnvVar = "OPERATOR_NAMESPACE_LOCAL"
```

Comment:
Why is one exported and the other not?

Reply:
`operatorCondEnvVar` will be used by OLM to set operator conditions name and `OperatorNsEnvVar` will be used by the users. But yeah, `OperatorNsEnvVar` need not be exported as its just an env variable. Modified it.